File: Resueltos_por_competidor/Alejandra_Carvajal/lift.py
# Autor: Alejandra Carvajal
# Problema: Lift
# Juez online: LightOJ
# Veredicto: Accepted
# URL: https://lightoj.com/problem/lift

import math
t = int(input())

for i in range(1, t+1):
    p, a = map(int, input().split())
    
    puerta = 3
    entrada_salida = 5
    
    f = abs(a - p) * 4
    ground_floor = p * 4
    
    tiempo = (puerta * 2) + (entrada_salida * 2) + f + ground_floor + puerta
    
    print(f"Case {i}: {tiempo}")
    

File: Resueltos_por_competidor/David_Velasquez/Lift.java
/*
 * Autor: DavsAlejo
 * Problema: Lift
 * Juez online: LigthOJ
 * Veredicto: Accepted
 * https://lightoj.com/problem/lift
 */

import java.util.Scanner;

public class Main {
	static Scanner src = new Scanner(System.in);

	public static void main(String[] args) {
		
		int T = src.nextInt();
	    String salida = "";

		for (int i = 1; i <= T; i++) {

			int piso = src.nextInt();
			int pisoDeAscensor = src.nextInt();

			int total = Math.abs(pisoDeAscensor-piso) * 4 + (piso*4) + 19 ;
			
			salida += "Case " + i + ":" + " " + total+"\n";
			
			
		}
		
		System.out.print(salida);
		

	}

}
